### DISCUSSION TITLE
Undo delete

### DISCUSSION THREAD

**Discussion by oldglusu** [adrcotfas/goodtime#332] [Q&A] [answered]
Under Statistics, I selected a session that I wanted to delete, but then accidentally deleted the last 9000 sessions or so. Can I undo this? Where is the data stored?

**Comment by adrcotfas** [comment, marked as answer]
Unfortunately there is no way to restore them unless you have a backup. The only protection for such cases is the alter dialog asking if you're sure you want to delete the selected sessions.

**Reply by oldglusu** [reply]
thx!
